src/middlewared/middlewared/plugins/enclosure_/slot_mappings.py

```diff
@@ -4,7 +4,26 @@
 # See the file LICENSE.IX for complete terms and conditions
 
 from .constants import SYSFS_SLOT_KEY, MAPPED_SLOT_KEY, SUPPORTS_IDENTIFY_KEY
-from .enums import ControllerModels, JbodModels
+from .enums import ControllerModels, JbodModels, JbofModels
+
+
+def get_jbof_slot_info(model):
+    """This function returns a dictionary that maps
+    nvme drives that are mounted via NVMe over Fabrics"""
+    if model == JbofModels.ES24N.name:
+        return {
+            'any_version': True,
+            'versions': {
+                'DEFAULT': {
+                    'model': {
+                        model: {
+                            i: {SYSFS_SLOT_KEY: i, MAPPED_SLOT_KEY: i, SUPPORTS_IDENTIFY_KEY: True}
+                            for i in range(1, 25)
```

Comment:
Just an observation.  SUPPORTS_IDENTIFY_KEY is True, but currently I don't see any code present in this PR to facilitate that.

Reply:
@bmeagherix right, I haven't added that part (yet). Will do so after this one.